antarest/study/storage/abstract_storage_service.py

```diff
@@ -315,14 +320,20 @@ def export_output(self, metadata: T, output_id: str, target: Path) -> Path:
         logger.info(f"Exporting output {output_id} from study {metadata.id}")
 
         path_output = Path(metadata.path) / "output" / output_id
-        if not path_output.exists():
+        path_output_zip = Path(metadata.path) / "output" / f"{output_id}.zip"
+
+        if path_output_zip.exists():
+            return path_output_zip
```

Comment:
the zip must be copied at the target path since the expected path of the zipped output is provided